incubator/jaeger/templates/agent-ds.yaml

```diff
@@ -18,14 +18,14 @@ spec:
     metadata:
       labels:
         app: "{{ template "jaeger.name" . }}"
-        component: "jaeger-agent"
+        component: "agent"
         release: "{{ .Release.Name }}"
         jaeger-infra: agent-instance
     spec:
       nodeSelector:
 {{ toYaml .Values.agent.nodeSelector | indent 8 }}
       containers:
-      - name: "jaeger-agent"
+      - name: "{{ template "jaeger.fullname" . }}-agent"
```

Comment:
You could have left the container names as they were. Feel free to change those back if you like.

Reply:
I would like to keep them with the jaeger.fullname so that if there is a situation where you want to deploy multiple independent instances, this gives you that flexibility for unique names.